src/demo/components/ComponentPage/ComponentPage.tsx

```diff
@@ -0,0 +1,137 @@
+import * as React from 'react';
+import { Button, ButtonType } from '../../../Button';
+import { css } from '../../../utilities/css';
+import {
+  Link
+} from '../../../index';
+import './ComponentPage.scss';
+
+export interface IComponentPageProps {
+  title: string;
+  componentName: string;
+  exampleCards: Array<JSX.Element>;
+  propertiesTables?: Array<JSX.Element>;
+  bestPractices?: JSX.Element;
+  dos?: JSX.Element;
+  donts?: JSX.Element;
+  overview: JSX.Element;
+  route: string;
+}
+
+export interface IComponentPageState {
+}
+
+export class ComponentPage extends React.Component<IComponentPageProps, IComponentPageState> {
+
+  constructor(props: IComponentPageProps) {
+    super(props);
+  }
+
+  public render() {
+
+    return (
+      <div className='ComponentPage'>
+        <div className={ this.props.componentName }>
+          <div className='ComponentPage-header'>
```

Comment:
The new Fabric site has its own PageHeader component, so if we pull in these pages we'll end up with duplicate headers. Maybe make a PageHeader component specific for the Fabric React project, which would be outside of the ComponentPage?